src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -162,6 +166,8 @@ class VtolAttitudeControl
 	struct vehicle_local_position_s		_local_pos;
 	struct airspeed_s 					_airspeed;			// airspeed
 	struct battery_status_s 			_batt_status; 		// battery status
+	struct vehicle_command_s			_vehicle_cmd;
+	struct vehicle_command_s			_vehicle_transition_cmd; // stores transition commands only
```

Comment:
Any last command gets stored in `_vehicle_cmd`, only transition commands get stored in `_vehicle_transition_command` (and that's the only member which is actually used). I think I'll remove the later and add a distinct member variable for the transition command request.

The `_vehicle_cmd` is actually never used, I could make it local in the polling function. Do you usually avoid this to not increase the stack size of those functions?